themes/bootstrap3/templates/layout/js-configs.phtml

```diff
@@ -0,0 +1,7 @@
+<?php /* Global context config only */
+
+//$this->jsConfigs()->addConfigPaths(
```

Comment:
I think that we still need a warning here not to expose any sensitive settings. Something like `WARNING: Any settings here are exposed to all users in plain text, so do not include any secrets here.`

Reply:
Perhaps also a couple more examples of the different ways `addConfigPaths` can be called?

Reply:
I added some explanation an the warning. 

I also added another example. Since this got change to a simple key value store I think two examples should be sufficient.